README.md

```diff
@@ -322,6 +322,10 @@ server {
         proxy_read_timeout 24h;
         proxy_http_version 1.1;
         proxy_set_header Connection "";
+        ## Remove comments only if you use argument USE_FORWARDED_HEADERS=1 ##
```

Comment:
It’s safe to always provide these headers, no need to comment them. So, the comment should be “Be sure to set USE_FORWRDED_HEADERS to 1 to allow the hub to use those headers”.

Reply:
Ok to change the comment with "Be sure to set USE_FORWARDED_HEADERS=1 to allow the hub to use those headers"